spec/lex.dd

```diff
@@ -600,7 +600,7 @@ $(GNAME DecimalInteger):
     $(GLINK NonZeroDigit) $(GLINK DecimalDigitsUS)
 
 $(GNAME BinaryInteger):
-    $(GLINK BinPrefix) $(GLINK BinaryDigitsUS)
+    $(GLINK BinPrefix) $(GLINK BinaryDigitsNoSingleUS)
```

Comment:
@thewilsonator this change looks completely unrelated.

Reply:
Ohh, good catch.

Reply:
Hmm i think I must have `git reset --soft HEAD~N` for `N` equals one more than I thought. I was fighting both `git` and the stupid whitespace checker a lot with this, that that happened is not really all that surprising.

Reply:
When you do changes to people's own branches be sure to make backups and then compare your changes with the backed up branch before force pushing.